### PR TITLE
use recaptcha.net

### PR DESCRIPTION
**Pull request by jhanifen**

### CODE REVIEW

**Comment by jhanifen** [comment]
Apologize, meant to commit this to our forked version.  Although this is a change that seems worth making for all users so the recaptcha works better globally per the Google FAQ listed here https://developers.google.com/recaptcha/docs/faq#can-i-use-recaptcha-globally

**Comment by grosser** [comment]
hmmm this should be read from the config ?

**Comment by grosser** [comment]
followup for https://github.com/ambethia/recaptcha/pull/320

File: demo/rails/app/views/captchas/index.html.erb [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <div id="example3"></div>
     <%= submit_tag %>
   <% end %>
-  <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
+  <script src="https://www.recaptcha.net/recaptcha/api.js?onload=onloadCallback&render=explicit" async defer></script>
```
Comment (grosser) [inline demo/rails/app/views/captchas/index.html.erb]:
```suggestion
  <script src="<%= Recaptcha.configuration.api_server_url %>?onload=onloadCallback&render=explicit" async defer></script>
```

Comment (jhanifen) [reply demo/rails/app/views/captchas/index.html.erb]: that would work great and makes a lot more sense.

**Comment by grosser** [comment]
5.0.1

**Comment by grosser** [comment]
derp ... this is just demo code, I yanked 5.0.1

**Pull request merged**